cfme/fixtures/cfme_data.py

```diff
@@ -4,5 +4,5 @@
 
 
 @pytest.fixture(scope="session")
```

Comment:
is this suppose to be session still?

Reply:
yes. I have to get rid of session scoped fixtures which use appliance. the rest can be left as is.